index.php

```diff
@@ -25,8 +25,7 @@
                         <th><A href="?sortby=0&sortorder=desc" style="color: white">Length</A></th>
                         <th><A href="?sortby=1&sortorder=desc" style="color: white">Switches</A></th>
                         <th><A href="?sortby=6&sortorder=desc" style="color: white">Trees</A></th>
-                        <th><A href="?sortby=2&sortorder=desc" style="color: white">Locos</A></th>
-                        <th><A href="?sortby=3&sortorder=desc" style="color: white">Carts</A></th>
+                        <th><A href="?sortby=6&sortorder=desc" style="color: white">Rolling Stock</th>
                         <th><A href="?sortby=4&sortorder=desc" style="color: white">Slope</A></th>
```

Comment:
sortby=6 means sort by trees.
you have to be able to sort by locos and carts
As I had trees and switches in one column, you asked me to have each value in its own column